EmployeeConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.ifour.EmployeeManagement.Employee;
+
+import org.springframework.beans.factory.annotation.Configurable;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+
+import java.util.List;
+
+@Configurable
+public class EmployeeConfig {
+    @Bean
+    CommandLineRunner commandLineRunner(EmployeeRepository repository)

```

Comment:
A quick question: why do we need command line runner? How are you planning to invoke this?